frb_dart/lib/src/ffi/ffi_web.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:async';
+
+import 'package:js/js.dart';
+export 'package:js/js.dart';
+import 'package:js/js_util.dart' show promiseToFuture;
+export 'package:js/js_util.dart' show promiseToFuture;
+
+import 'dart:typed_data';
+import '../ffi.dart' show WasmModule;
+
+typedef ExternalLibrary = FutureOr<WasmModule>;
+
+@JS()
+external bool? get crossOriginIsolated;
+
+@JS('Function')
+class _UnaryFunction {
```

Comment:
uber-nit: Why "unary" here, since the function does not seem to have any arguments at all... Maybe `_Function` or `_JsFunction` is enough?